net/wireguard/src/opnsense/mvc/app/controllers/OPNsense/Wireguard/Api/ServiceController.php

```diff
@@ -52,7 +53,20 @@ class ServiceController extends ApiMutableServiceControllerBase
     public function showconfAction()
     {
         $backend = new Backend();
-        $response = $backend->configdRun("wireguard showconf");
+        $response_org = $backend->configdRun("wireguard showconf");
+        $response = '';
+
+        $pubnames = $this->getPubkeyNames();
+        $rp_lines = preg_split('/\r\n|\r|\n/', $response_org);
+        foreach($rp_lines as $line)
```

Comment:
not PSR12 style as well as a lot of other lines

Reply:
changed to PSR12, i hate the PSR12 stlye ;-) , my main programming language is c# , there is no PSR12 in use.